script/licenses.py

```diff
@@ -244,8 +242,92 @@ def main(argv: Sequence[str] | None = None) -> int:
     return exit_code
 
 
+def extract_licenses(args: ExtractArgs) -> int:
+    """Extract license data for installed packages."""
+    licenses = sorted(
+        [get_package_metadata(dist) for dist in list(metadata.distributions())],
+        key=lambda dist: dist["name"],
+    )
+    Path(args.output_file).write_text(json.dumps(licenses, indent=2))
+    return 0
+
+
+def get_package_metadata(dist: metadata.Distribution) -> PackageMetadata:
+    """Get package metadata for distribution."""
+    return {
+        "name": dist.name,
+        "version": dist.version,
+        "license_expression": dist.metadata.get("License-Expression"),
+        "license_metadata": dist.metadata.get("License"),
+        "license_classifier": extract_license_classifier(
+            dist.metadata.get_all("Classifier")
+        ),
```

Comment:
Changed the metadata attribute names to camelcase to match the official core metadata spec spelling (`license-expression` -> `License-Expression`). It doesn't really matter in the end as `.get` uses `name.lower()` when checking the attribute. So both spellings work fine.

https://packaging.python.org/en/latest/specifications/core-metadata/#license-expression